api/host.go

```diff
@@ -42,6 +43,8 @@ type (
 		MinRecentScanFailures uint64    `json:"minRecentScanFailures"`
 	}
 
+	// SearchHostsRequest is the request type for the /api/bus/search/hosts
```

Comment:
This type is currently abused a bit, it's used by the `autopilot` and then reused in the `bus` but we just ignore the `usabilityMode`. That's a little unfortunate because we'll have to just add an autopilot ID and validate both parameters are used correctly.